### PR TITLE
Issue312 accd as list

### LINKED ISSUE #312: ACDD attributes provided as lists will cause error
An example is

`string :platform = "&quot;Basis&quot;: &quot;Space-based Platforms&quot;", " &quot;Category&quot;: &quot;Earth Observation Satellites&quot;", " &quot;Sub_Category&quot;: &quot;Sentinel-1&quot;", " &quot;Short_Name&quot;: &quot;Sentinel-1A&quot;", " &quot;Long_Name&quot;: &quot;Sentinel-1A&quot;" ;`

in Sentinel-1 datasets from NBS.

### PR DESCRIPTION
**Pull request by mortenwh**
**Summary**: closes #312 - this is just an added test. The code behaves correctly.

**Related issue**: 

**Suggested reviewer(s)**:

**Reviewer checklist**:

- [ ] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
- [ ] 100% test coverage of new code - meaning:
    - [ ] The overall test coverage increased or remained the same as before
    - [ ] Every function is accompanied with a test suite
    - [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
    - [ ] Functions with optional arguments have separate tests for all options
- [ ] Examples are supported by doctests
- [ ] All tests are passing
- [ ] All names (e.g., files, classes, functions, variables) are explicit
- [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: requirements.txt
Comment (Teddy-1000) [inline requirements.txt:1]: Why was this file added? Requirements should be kept in pyproject.toml. This adds duplicate files where we need to maintain 2 files with requirements.

**Review by Teddy-1000 — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (mortenwh) [reply requirements.txt:1]: for easy installation in a virtualenv (`pip install -r ...`)

Comment (Teddy-1000) [reply requirements.txt:1]: This is still easy to install in a venv with `pip install . --editable` from the top level in the repo, this will also add script paths to the venv. Leaving requirements superfluous. I also recommend not having 2 files for tracking dependencies.

Comment (mortenwh) [reply requirements.txt:1]: does it work the same way as `pip install -r ...`? Otherwise, this is not a big deal. That extra file is just a helpful addition that doesn't harm anyone, even if it gets outdated. Apart from that, I agree that repetition is not a good thing.

Comment (Teddy-1000) [reply requirements.txt:1]: It works the same way as `pip install -r ...`, if you just want the package installed and not going to do any development on the software you can also skip the `--editable`. 
I do disagree with the view on the extra file. If it gets outdated and someone tries to use it, they will not know why the software is broken. If you want to keep the file, an extra CI pipeline using the requirements.txt  for testing should be added.

Comment (mortenwh) [reply requirements.txt:1]: ok - I have removed the file and updated the README. Better?

**Review by Teddy-1000 — APPROVED** [review]
LGTM
